templeton/src/main/java/edu/virginia/psyc/templeton/persistence/HelpSeeking.java

```diff
@@ -1,20 +1,25 @@
 package edu.virginia.psyc.templeton.persistence;
 
-import org.mindtrails.domain.questionnaire.SecureQuestionnaireData;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
+import org.mindtrails.domain.questionnaire.SecureQuestionnaireData;
 
 import javax.persistence.Entity;
 import javax.persistence.Table;
 
-/**
```

Comment:
Thank you for cleaning up the comments.  My IntelliJ auto-inserts these and I'm lazy about cleaning them up.   This file is very concise and clean.  Well done.